aiida/control/postgres.py

```diff
@@ -24,7 +25,9 @@
 
 _CREATE_USER_COMMAND = 'CREATE USER "{}" WITH PASSWORD \'{}\''
 _DROP_USER_COMMAND = 'DROP USER "{}"'
-_CREATE_DB_COMMAND = 'CREATE DATABASE "{}" OWNER "{}"'
+_CREATE_DB_COMMAND = ('CREATE DATABASE "{}" OWNER "{}" ENCODING \'UTF8\' '
+                      'LC_COLLATE=\'en_US.UTF-8\' LC_CTYPE=\'en_US.UTF-8\' '
+                      'TEMPLATE=template0')
```

Comment:
about this one I wasn't entirely sure.
perhaps it's safer to revert this since all the utf8-logic isn't in 0.12

Reply:
I agree, if it is not necessary in terms of fixing the sudo bug, I would omit it for consistency